pkgs/build-support/kernel/make-initrd-ng.nix

```diff
@@ -74,10 +74,9 @@ in
   passAsFile = ["contents"];
   contents = lib.concatMapStringsSep "\n" ({ object, symlink, ... }: "${object}\n${if symlink == null then "" else symlink}") contents + "\n";
 
-  nativeBuildInputs = [makeInitrdNGTool patchelf cpio] ++ lib.optional makeUInitrd ubootTools ++ lib.optional strip binutils;
+  nativeBuildInputs = [makeInitrdNGTool cpio] ++ lib.optional makeUInitrd ubootTools ++ lib.optional strip binutils;
 
-} // lib.optionalAttrs strip {
-  STRIP = "${(binutils.nativeDrv or binutils).targetPrefix}strip";
+  STRIP = lib.optionalString strip "${(binutils.nativeDrv or binutils).targetPrefix}strip";
```

Comment:
As I said, this isn't right. It can't be an empty string. It has to be `null`.

Reply:
Ugh. Forgot that's not what `optionalString` does.